test/inductor/test_torchinductor.py

```diff
@@ -733,9 +733,13 @@ class SweepInputs2:
         "broadcast1",
         "broadcast2",
         "broadcast3",
-        "double",
         "int",
     ]
+
+    # MPS doesn't support 64-bits floating point
+    if GPU_TYPE != "mps":
+        input_gen_types1.append("double")
```

Comment:
I'm slightly troubled by this change, as `GPU_TYPE` is a global variable and it will always returns "mps" on Mac platform, so it will skip double test when it's instantiated for both CPU and GPU.
Perhaps the better approach would be to `raise unittest.SkipTest("double not supported on {device}")` when this test is run. Let me propose a change that will do something of the nature for different subset of tests..